crates/rpc/rpc-types-util/src/transaction/mod.rs

```diff
@@ -0,0 +1,177 @@
+
+use reth_primitives::{
+    AccessListItem, Address, BlockNumber, Bytes, Transaction as PrimitiveTransaction,
+    TransactionKind as PrimitiveTransactionKind, TransactionSignedEcRecovered, TxType, H256, U128,
+    U256, U64,
+};
+
+use reth_rpc_types::Signature;
+
+use serde::{Deserialize, Serialize};
+
+/// Transaction object used in RPC
+#[derive(Debug, Default, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Transaction {
```

Comment:
why was this type moved?
this is an rpc type and should remain in rpc-types

Reply:
Right

Reply:
Also, I copied the type , not moved .